aptos-move/e2e-testsuite/src/tests/validator_set_management.rs

```diff
@@ -1,431 +0,0 @@
-// Copyright (c) Aptos
-// SPDX-License-Identifier: Apache-2.0
-
-use aptos_transaction_builder::stdlib::*;
-use aptos_types::{
-    on_chain_config::{new_epoch_event_key, VMPublishingOption},
-    transaction::{TransactionOutput, TransactionStatus},
-    vm_status::KeptVMStatus,
-};
-use language_e2e_tests::{
-    account::Account, current_function_name, executor::FakeExecutor, test_with_different_versions,
```

Comment:
why isn't this relevant? it feels a bit weird deleting it, since we still use this framework, no?

Reply:
we don't expose those scripts, and we'll probably have different scripts